### PR TITLE
Adobe Analytics

### PR DESCRIPTION
**Pull request by TishamIslam**
### Description
Added Adobe Analytics to the project.

### Addtional Notes
Not sure who to talk to about these bits:
image
(Currently set to "ESDC_GC" )

image
(Currently set to "ESDC-EDSC_VA-AV" )

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,9 +24,15 @@
     <meta property="og:title" content="Virtual Assistant | Assistant Virtuel" />
     <meta property="og:description" content="Virtual Assistant | Assistant Virtuel" />
 
+    <!-- dcterms metadata -->
+    <meta name="dcterms.creator" content="ESDC_GC"/>
```
Comment (nibivi77) [inline index.html:28]: I imagine we should follow what Alpha did: have a locale for the content.

en: Employment Social Development Canada.

fr: Emploi et Développement social Canada

File: index.html [hunk 2]
```diff
@@ -23,9 +24,15 @@
...
+    <meta name="dcterms.accessRights" content="2"/> 
+    <meta name="dcterms.service" content="ESDC-VA_MSCA-AV"/>
```
Comment (nibivi77) [inline index.html:30]: Following Alpha, the content should probably be : ESDC-EDSC_VA-AV.

**Review by nibivi77 — COMMENTED** [review]
I noticed that Alpha also added a dcterms.title and a dcterms.language.

Check out their index.js inside the pages folder.

File: index.html (hunk 2 above)
Comment (TishamIslam) [reply index.html:30]: I probably should've checked what the french version of ESDC was, my bad 😅

File: index.html (hunk 1 above)
Comment (TishamIslam) [reply index.html:28]: I'm not sure how we'd add a locale for the index page since it's not a Vue file...

**Comment by TishamIslam** [comment]
In regards to e79/80 Form Submission, we have no form elements so it doesn't apply but we're still handling information, not sure what there should be done about it/if something should be done about it.
image

File: index.html (hunk 1 above)
Comment (nibivi77) [reply index.html:30]: Hummm good question.

I found this link that could help: https://www.epiloge.com/how-to-add-dynamic-meta-tags-to-your-vuejs-app-for-google-seo-0fa058

Comment (TishamIslam) [reply index.html:30]: vue-meta doesn't have vue 3 support and even though the latest alpha version does, it doesn't seem to have any documentation. I modified the meta tag directly with JS instead for now.

**Comment by TishamIslam** [comment]
We have to wait on them adding Adobe Analytics for our domain before we can do further testing, but I think everything we need is here.

**Review by nibivi77 — APPROVED** [review]
Looks good to me Tisham!

**Pull request merged**